engines/mxnet/mxnet-engine/src/main/java/ai/djl/mxnet/jna/JnaUtils.java

```diff
@@ -1890,6 +1890,11 @@ public static CachedOp createCachedOp(
         String[] keys = {"data_indices", "param_indices", "static_alloc", "static_shape"};
         String[] values = {dataIndices.values().toString(), paramIndices.toString(), "1", "1"};
 
+        if (Boolean.parseBoolean(System.getenv("MXNET_DISABLE_STATIC"))) {
```

Comment:
should we define two variables:
MXNET_STATIC_ALLOC=0
MXNET_STATIC_SHAPE=0

Reply:
These usually comes together, there is no options to set just 1